schemas/layer-2.cue

```diff
@@ -1,84 +1,131 @@
 package schemas
 @go(layer2)
 
+// Layer2 describes a collection or catalog of technology-specific, threat-informed security controls
+// that can be applied to an information system.
 #Layer2: {
+    // metadata about the Layer2 collection
     metadata?: #Metadata
-
+    // one or more ControlFamily objects
     "control-families"?: [...#ControlFamily] @go(ControlFamilies)
+    // zero or more Threats that are mitigated by the controls in the ControlFamilies
     threats?: [...#Threat] @go(Threats)
+    // zero or more Capabilities in use that inform the controls in the ControlFamilies
     capabilities?: [...#Capability] @go(Capabilities)
-
+    // zero or more mapping references to controls defined in this or other Layer2 frameworks or standards
     "shared-controls"?: [...#Mapping] @go(SharedControls)
+    // zero or more mapping references to threats defined in this or other Layer2 frameworks or standards
     "shared-threats"?: [...#Mapping] @go(SharedThreats)
+    // zero or more mapping references to capabilities defined in this or other Layer2 frameworks or standards
     "shared-capabilities"?: [...#Mapping] @go(SharedCapabilities)
 }
 
-// Resuable types //
-
 #Metadata: {
+    // unique identifier for the Layer2 collection
     id: string
+    // name for the Layer2 collection
     title: string
+    // description of the Layer2 collection
     description: string
     version?: string
+    // timestamp of the last modification to the Layer2 collection
     "last-modified"?: string @go(LastModified)
+    // Applicability categories are facets that apply to the controls in this Layer2 collection. These facet values can be used to filter which controls from Layer2 are used in Layer 3 or Layer 4.
     "applicability-categories"?: [...#Category] @go(ApplicabilityCategories)
```

Comment:
For the OSPS Baseline, I _think_ this could look something like:

```yaml
applicability-categories:
  - id: level1
    name: Level 1
    description: for any code or non-code project with any number of maintainers or users
  - id: level2
    name: Level 2
    description: for any code project that has at least 2 maintainers and a small number of consistent users
  - id: level3
    name: Level 3
    description: for any code project that has a large number of consistent users
```

Reply:
Yep. or snake/kebab, whatever.